daft/execution/execution_step.py

```diff
@@ -695,7 +693,7 @@ def _multislice(self, inputs: list[Table]) -> list[Table]:
             assert start >= 0, f"start must be positive, but got {start}"
             end = min(end, len(input))
 
-            indices_block = Series.from_numpy(np.arange(start, end))
+            indices_block = Series.from_numpy(list(range(start, end)))
```

Comment:
this should be `from_pylist` since the input is a pylist. Also note we have a DataArray::arange but it not exposed to Series or Python yet.